### PR TITLE
Increasing verbosity of some of our logs

### PR DESCRIPTION
**Pull request by petderek**
### Summary

Some of our logs around the task lifecycle are only logged at debug  level.
This can make identifying problems a difficult task unless debug logs are
provided. This is an attempt to make our logs more useful when ran with
default settings.

### Licensing

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: agent/api/task/task.go [hunk 1]
```diff
@@ -720,8 +720,6 @@ func (task *Task) updateTaskKnownStatus() (newStatus apitaskstatus.TaskStatus) {
 			containerEarliestKnownStatus.String(), task)
 		return apitaskstatus.TaskStatusNone
 	}
-	seelog.Debugf("Container with earliest known container is [%s] for task: %s",
```
Comment (fenxiong) [inline agent/api/task/task.go]: why is this deleted?

Comment (adnxn) [reply agent/api/task/task.go]: @petderek: lol why did we do this again?

Comment (petderek) [reply agent/api/task/task.go]: Its duplicated elsewhere.

Comment (petderek) [reply agent/api/task/task.go]: er wait, no its not. I'll re-add this

**Comment by adnxn** [comment]
that's weird, havent seen this unit test is fail `TestHandlerReconnectsOnDiscoverPollEndpointError`

**Comment by adnxn** [comment]
closing for now - we'll revisit if required.
